src/screens/PinScreen.js

```diff
@@ -44,6 +46,8 @@ const mapDispatchToProps = (dispatch) => ({
   setLoadHistoryStatus: (active, error) => dispatch(setLoadHistoryStatus(active, error)),
   setInitWallet: (words, pin) => dispatch(setInitWallet(words, pin)),
   setTempPin: (pin) => dispatch(setTempPin(pin)),
+  startWalletSuccess: () => dispatch(startWalletSuccess()),
```

Comment:
We don't need this anymore

Reply:
You are right, thanks

Fixed in e9c92f356c017d00d4923fef3b442439d2505433